core-bundle/tests/Framework/ContaoFrameworkTest.php

```diff
@@ -634,6 +635,11 @@ public function testRegistersTheHookServices(): void
         );
     }
 
+    public function testServiceIsResetable(): void
+    {
+        $this->assertInstanceOf(ResetInterface::class, $this->mockFramework());
```

Comment:
You should test an actual reset, shouldn't you?

Reply:
added in 5ba4e1ec97fc5d592da43045402a34718b9ccb14